sklearn/preprocessing/tests/test_knn_imputation.py

```diff
@@ -0,0 +1,432 @@
+from __future__ import division
```

Comment:
I'd also rather this code in test_imputation, so that:
* it's easy to find
* we can more easily refactor some of the common tests between KNNImputer and Imputer (now or eventually)